client/me/purchases/billing-history/utils.jsx

```diff
@@ -62,15 +63,25 @@ export function renderTransactionAmount( transaction, { translate, addingTax = f
 		return transaction.amount;
 	}
 
+	const countryTaxInfo =
+		translate instanceof Function &&
+		getVatVendorInfo( transaction.tax_country_code, transaction.date, translate )
+			? getVatVendorInfo( transaction.tax_country_code, transaction.date, translate )
+			: { taxName: 'tax' };
+
 	const taxAmount = addingTax
-		? translate( '(+%(taxAmount)s tax)', {
+		? translate( '(+%(taxAmount)s ', {
 				args: { taxAmount: transaction.tax },
 				comment: 'taxAmount is a localized price, like $12.34',
-		  } )
-		: translate( '(includes %(taxAmount)s tax)', {
+		  } ) +
+		  countryTaxInfo.taxName +
```

Comment:
Joining translated strings is not wise because different languages might place the words in a different order. To include a variable inside a translated string you can use placeholders. There's different ways to implement them, but in this case, probably a string with a percent sign followed by a label and then an `s` like this:

```js
translate( '(+%(taxAmount)s %(taxName)s)', { args: { taxAmount: transaction.tax, taxName: countryTaxInfo.taxName } } )
```